src/Host/Properties/launchSettings.json

```diff
@@ -16,7 +16,7 @@
       "launchUrl": "swagger",
       "applicationUrl": "https://localhost:5001;http://localhost:5000",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "NotDevelopment"
```

Comment:
This is temporary to avoid [this bug](https://github.com/jbogard/MediatR/issues/707) which is being investigated [here](https://github.com/dotnet/runtime/issues/65145) (is only when environment is set to "Development").